ansible/playbooks/roles/upgrade/tasks/opensearch.yml

```diff
@@ -0,0 +1,70 @@
+---
+- name: OpenSearch | Get information about installed packages as facts
+  package_facts:
+    manager: auto
+  when: ansible_facts.packages is undefined
+
+- name: OpenSearch | Assert that elasticsearch-oss package is installed
+  assert:
+    that: ansible_facts.packages['elasticsearch-oss'] is defined
+    fail_msg: elasticsearch-oss package not found, nothing to upgrade
+    quiet: true
+
+- name: OpenSearch | Include defaults from OpenSearch role
+  include_vars:
+    file: roles/opensearch/defaults/main.yml
+    name: opensearch_defaults
+
+- name: OpenSearch | Include vars from opensearch role  # requires epicli upgrade -f <manifest>.yml
+  include_vars:
+    file: roles/opensearch/vars/main.yml
+    name: opensearch_vars
+
+- name: OpenSearch | Ensure Opensearch service user exists
+  user:
+    name: "{{ opensearch_vars.specification.opensearch_user }}"
+    state: present
+    shell: /bin/bash
+
+- name: OpenSearch | Ensure directory structure exists
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: "{{ opensearch_vars.specification.opensearch_user }}"
+    group: "{{ opensearch_vars.specification.opensearch_user }}"
```

Comment:
Specify in variable that this is an os user? Maybe also separate group?

Reply:
ad a)
ok
Implemented now.
ad b) 
As discussed before: there are two approaches and it's hard to say which is better.
Probably would be good to discuss it within the team before changing.